miio/chuangmi_camera.py

```diff
@@ -0,0 +1,301 @@
+""" Xiaomi Chuangmi camera (chuangmi.camera.ipc009) support. """
+
+import logging
+from typing import Any, Dict
+
+from .click_common import command, format_output
+from .device import Device
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CameraStatus:
+    """Container for status reports from the Xiaomi Chuangmi Camera."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    @property
+    def power(self) -> str:
+        """ Camera power """
+        return self.data["power"]
```

Comment:
Thanks for review! 
>If yes, maybe check for the wanted value and return boolean instead of str.
Got it, will figure how to do it (I think I saw something like that in some other device class) and fix.

>If yes, maybe check for the wanted value and return boolean instead of str.
Sure, will fix this as well.